internal/controller/preflight_checks.go

```diff
@@ -40,9 +43,10 @@ const (
 var (
 	moreThanOneCoreProviderInstanceExistsMessage = "CoreProvider already exists in the cluster. Only one is allowed."
 	moreThanOneProviderInstanceExistsMessage     = "There is already a %s with name %s in the cluster. Only one is allowed."
-	capiVersionIncompatibilityMessage            = "capi operator is only compatible with %s providers, detected %s for provider %s."
-	waitingForCoreProviderReadyMessage           = "waiting for the core provider to install."
-	emptyVersionMessage                          = "version cannot be empty"
+	capiVersionIncompatibilityMessage            = "CAPI operator is only compatible with %s providers, detected %s for provider %s."
+	invalidGithubTokenMessage                    = "Invalid github token, please check your github token value and it's permissions"
```

Comment:
should we specify the permission scope (repo?) to let know the user what permissions are really needed?

Reply:
Yes, I will add it to the docs. It is the same as clusterctl, only access to the repositories.